src/config.rs

```diff
@@ -557,6 +559,9 @@ impl Tunables {
             notifications: self.notifications.or(other.notifications),
             image_preview: self.image_preview.or(other.image_preview),
             user_gutter_width: self.user_gutter_width.or(other.user_gutter_width),
+            external_edit_file_suffix: self
+                .external_edit_file_suffix
+                .or(other.external_edit_file_suffix),
```

Comment:
I would be fine with an `.unwrap_or_else(|| ".md".into())` here. iamb interprets the text as markdown, so that's probably the most useful default.